datafusion/core/src/physical_plan/file_format/parquet/page_filter.rs

```diff
@@ -419,10 +468,37 @@ macro_rules! get_min_max_values_for_page_index {
                     vec.iter().map(|x| x.$func().cloned()),
                 )))
             }
-            Index::INT96(_) | Index::BYTE_ARRAY(_) | Index::FIXED_LEN_BYTE_ARRAY(_) => {
+            Index::BYTE_ARRAY(index) => {
+                let vec = &index.indexes;
```

Comment:
decimal should be supported for this logical type.

Reply:
Arrow-rs contains the method of decoding decimal from byte array in `ByteArrayReader`

Reply:
Thanks, i prefer align with row group, do them together in other pr.